components/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: single
+author_profile: false
+title: Zarr Components
+sidebar:
+  title: "Components"
+  nav: sidebar
+---
+
+Zarr consists of several components, both abstract and concrete. 
+These span both the physical storage layer and the conceptual structural layer. 
+Zarr-related projects all follow the Zarr specification (and hence data model), but otherwise may choose to implement other layers however they wish.
+
+## Abstract components
+
+These abstract components together describe what type of data can be stored in zarr, and how to store it, without assuming you are working in a particular programming language, or with a particular storage system.
+
+**Specification**: All zarr-related projects obey the [Zarr Specification](https://zarr-specs.readthedocs.io/), which formally describes how to serialize and de-serialize array data as byte streams as well as store metadata via an [Abstract Key-Value Store Interface](https://zarr-specs.readthedocs.io/en/latest/v3/core/v3.0.html#abstract-store-interface). 
```

Comment:
It might be more accurate to call this the "Zarr Protocol". The protocol has a specification.

Reply:
I edited this - the more I think about it the more I think that the spec itself should explicit talk about the protocol and the format as separate things.

Reply:
See https://github.com/zarr-developers/zarr-developers.github.io/pull/131#issuecomment-2781516383 for more explanation